### PR TITLE
Refactor tests to Pest

### PR DESCRIPTION
**Pull request by alexmanase**

### CODE REVIEW

**Comment by freekmurze** [comment]
Seems like the tests are failing, could you take a look?

**Comment by alexmanase** [comment]
I'll look into it.

File: tests/PdfToTextTest.php [hunk 1]
```diff
@@ -10,7 +10,12 @@
 beforeEach(function () {
     $this->dummyPdf = __DIR__ . '/testfiles/dummy.pdf';
     $this->dummyPdfText = 'This is a dummy PDF';
-    $this->pdftotextPath = '/opt/homebrew/bin/pdftotext';
+
+    if (PHP_OS === 'Linux') {
```
Comment (freekmurze) [inline tests/PdfToTextTest.php:14]: I don't like `else` statements. Could you refactor this to a ternary?

**Review by freekmurze — CHANGES_REQUESTED** [review]

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**